docs/release/notes.md

```diff
@@ -41,7 +41,7 @@ To update to the OSG 3.5 series, please consult the page on
 
 | Version                                   | Date       | Summary                                                                 |
 |:------------------------------------------|:-----------|:------------------------------------------------------------------------|
-| [3.5.8-3](/release/3.5/release-3-5-8-3)   | 2020-01-29 | High Priority Release: IGTF 1.104                                       |
+| [3.5.8-3](/release/3.5/release-3-5-8-3)   | 2020-01-29 | High Priority Release: IGTF 1.104 pre-release                           |
```

Comment:
```suggestion
| [3.5.8-3](/release/3.5/release-3-5-8-3)   | 2020-01-29 | High Priority Release: OSG CA certs based on IGTF 1.104 pre-release                           |
```
the current phrasing makes it seem like we're releasing the IGTF CA certs as well